src/main/resources/db/migration/V1_7__create_visit_restriction_column.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE visit
+    ADD COLUMN visit_restriction   VARCHAR(80)
```

Comment:
@Column(nullable = false)

Reply:
This is for migrating the data - can condense now and drop the tables or wait until we have more changes.

Reply:
I would use original create visit table

Reply:
Will update